packages/two_percent/db/migrate/20250418202816_create_two_percent_phone_numbers.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateTwoPercentPhoneNumbers < ActiveRecord::Migration[8.0]
```

Comment:
When I ran the migrations in connect-server, it failed and I had to reduce the version to 
```suggestion
class CreateTwoPercentPhoneNumbers < ActiveRecord::Migration[7.1]
```
for all migrations to get them to run successfully. Is 7.1 our minimum supported Rails version?

Reply:
@phylum good point. I'll fix this